src/cascadia/TerminalApp/App.xaml

```diff
@@ -1,51 +1,59 @@
-<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under

```

Comment:
Weird that these xaml files keep getting reformatted by your editor - mind reverting them?